src/content/configuration/dev-server.md

```diff
@@ -836,6 +836,19 @@ module.exports = {
   }
 };
 ```
+The origin of the host header is kept when proxying by default, you can set `changeOrigin` to `true` to override it, this is useful in some situations like using [name-based virtual hosted sites](https://en.wikipedia.org/wiki/Virtual_hosting#Name-based).
```

Comment:
i suggest to rephrase a bit:
you can set `changeOrigin` to `true` to override it, ... **=>** you can set `changeOrigin` to `true` to override this behaviour. It is useful in some cases like ...

It is not exactly a situation but a use case, so a full stop before that sentence sounds better to me.